clang/test/CIR/CodeGen/ssync-fetch-and-add.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o - | FileCheck %s 
+
+#include <stdint.h>
+
+void foo(int64_t x) {
+  __sync_fetch_and_add(&x, 1);
```

Comment:
Instead of the new testfiles, incorporate this with other __sync_fetch_and_add testing in  `clang/test/CIR/CodeGen/atomic.cpp`